packages/odyssey-react/src/components/Infobox/Infobox.module.scss

```diff
@@ -20,21 +20,23 @@
     ". actions";
   grid-template-columns: max-content 1fr;
   width: 100%;
-  max-width: calc(#{$max-line-length} + #{$spacing-m} + #{$spacing-m});
-  padding-block: $spacing-m;
-  padding-inline: $spacing-m;
-  column-gap: $spacing-s;
+  max-width: calc(
+    var(--FontLineLengthMax) + var(--RootPadding) + var(--RootPadding)
+  );
+  padding-block: var(--RootPadding);
```

Comment:
I think we're aligned here re: variants. Though, you should double-check my naming below for the variant Background Colors.

`Root` here is intended to indicate the component part / scope of the padding rather than it being the default. Ex: `RootMargin` vs `ContentMargin`. 

Single-part components [like Checkbox](https://github.com/okta/odyssey/pull/1253) probably don't need this distinction, but it felt odd having a no-context property in here. "Where does this padding apply?"

I find "Root" odd too, though. With `Tooltip`, it felt natural to separate `Tip` and `Tail`.

@arnoldsandoval-okta suggested `InfoboxPadding` as a pattern, with the component itself representing the outermost container. Thoughts?

Reply:
@edburyenegren-okta thank you! let me start with your second question first: we should avoid referencing the component name here at all as a scope. It will make things more confusing because our run time API **already** references the component being themed:

```
theme={{
  [Infobox.theme]: { SpacePaddingBlock: "..." }
}} 
```
Our tools also take care of ensure values are unique even if they are used in two component themes.

I think this speaks to the first part of your comment about what modifier or context do we need to provide when styles are targeting the outmost container. I think keeping this terse is the way to go but I could be convinced.

Reply:
I'm good to go with no scope modifier for parents as long as we ensure that component part names are scoped:

```
MarginEnd: ...

ContentMarginEnd: ...
```

If folks are confused without something like Base/Root, I'm sure they'll let us know 😂

Reply:
@edburyenegren-okta ok cool let me fix up my Button PR!